src/App.test.js

```diff
@@ -28,7 +28,9 @@ describe('App', () => {
       usStates: {},
       vistedParks: [],
       currentParksToShow: [],
-      bucketListParks: []
+      bucketListParks: [],
+      pageStatus: 'landing',
+      // need to check for randomImageClass - not sure how to test a random number in state.
```

Comment:
@brittanystoroz We have some questions on testing for this Component. In our App component we set a random background image based on a random class on page load (see line 17 in App.js). Not sure how to test a random class name in our snapshot or in our default state on our App component. Currently these tests are both failing because the class name is random and changes each time the page is loaded.